.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
+sudo: false
+cache:
+  directories:
+    - node_modules
```

Comment:
Building should be a bit faster in cases of many dependencies, where taking an archived folder would be faster than (downloading and) extracting them individually. ParseDbf only has few dependencies so the advantage is not so big here.

Indeed, caches are not invalidated in a smart way by updating dependency references, but can be cleared explicitly via api or on https://travis-ci.org/stefanb/parseDBF/caches
full documentation: http://docs.travis-ci.com/user/caching/